packages/gatsby/gatsby-config.js

```diff
@@ -34,6 +34,14 @@ module.exports = {
   plugins: [
     `gatsby-plugin-emotion`,
     `gatsby-plugin-react-helmet`,
+    {
+      resolve: `gatsby-plugin-algolia-docsearch`,
+      options: {
+        apiKey: `029f65f2c00301615fd14958b67d6730`,
+        indexName: `yarnpkg_next`,
+        inputSelector: `.docsearch`,
```

Comment:
You can give two different classes maybe, it should accept two selectors I think

Reply:
From the look of it the script makes a `this.input = DocSearch.getInputFromSelector(inputSelector)`, so only one input 🤔